src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -196,7 +203,7 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
     setValue(SendFormFields.AmountFieldError, '')
 
     if (feeAsset.assetId !== assetId) {
-      setValue(SendFormFields.CryptoAmount, cryptoHumanBalance.toPrecision())
+      setValue(SendFormFields.AmountCryptoPrecision, cryptoHumanBalance.toPrecision())
```

Comment:
`cryptoHumanBalance` too?

Reply:
Also wondering if `toPrecision()` is needed here since this is coming from `selectPortfolioCryptoPrecisionBalanceByFilter` which should already be a crypto precision amount?

Reply:
Fixed `cryptoHumanBalance` in https://github.com/shapeshift/web/pull/6285/commits/52403fbb9b8f0f1afb7ae9ac22685a0bb453e8cb.

I left the `toPrecision()` as I don't want to risk any functional changes in this refactor.